__tests__/integration/examples/properties/known_types.cs

```diff
@@ -1,6 +1,8 @@
 using System;
 using LeanCode.Contracts;
 
+public class Query : IQuery<DateTimeOffset> {}
```

Comment:
I think name of this query should be something different than `Query` because later we have `Query extends Query`.

Reply:
You're right, I'll rename it.